VascularFlow/Initialization/InitializeFlowArrays.py

```diff
@@ -0,0 +1,26 @@
+import numpy as np
+
+
+def initialize_flow_arrays(nb_nodes: int):
+    """
+    Initializes flow-related arrays based on the spatial grid x_n.
+
+    Parameters:
+        nb_nodes (int): Number of spatial grid points.
+
+    Returns:
+        dict: A dictionary containing initialized arrays.
+    """
+    size = nb_nodes
+    return {
+        "h_n_1": np.ones(size),
```

Comment:
Do you actually need this many auxiliary fields? Can you put comments here explaining what each of these fields stores?